components/Snackbar/examples/SnackbarSimpleExample.m

```diff
@@ -27,6 +27,13 @@ @implementation SnackbarSimpleExample {
 
 - (void)viewDidLoad {
   [super viewDidLoad];
+  if (!self.colorScheme) {
+    self.colorScheme = [[MDCSemanticColorScheme alloc] init];
+  }
+  if (!self.typographyScheme) {
+    self.typographyScheme = [[MDCTypographyScheme alloc] init];
+  }
+  
```

Comment:
Consider breaking this out to a separate change.

Reply:
Reverted.